iac-extensions/docker/src/main/java/org/sonar/iac/docker/checks/utils/StringPredicate.java

```diff
@@ -30,6 +30,17 @@ public static Predicate<String> startsWithIgnoreQuotes(String value) {
     return str -> stripQuotes(str).startsWith(value);
   }
 
+  public static Predicate<String> startsWithIgnoreQuotes(String... values) {
```

Comment:
Wouldn't it be possible to reuse the startsWithIgnoreQuotes(String value) and chain it with `Or`?

Reply:
Yes, it is possible. However, most of the methods in the Command Detector API have the `List` or varargs variants.